README.md

```diff
@@ -1,2 +1,54 @@
-# admin.orders.api.ch.gov.uk
-Internal Admin web application tool used to examine orders.
+# admin.orders.web.ch.gov.uk
+
+## Summary
+
+* Internal Admin web application tool used to examine orders.
```

Comment:
The application will be accessible from outside of Companies House, so let's describe this project as "Web application used to examine and inspect orders."

Reply:
Resolved in latest commit.